### DISCUSSION TITLE
Tabs do not show differently from spaces in workflow editor

### DISCUSSION THREAD

**Discussion by elibroftw** [community/community#38455] [Actions]
Hi, so I was indenting a comment in my workflow, and wen I saved it, I got this syntax error:
![image](https://user-images.githubusercontent.com/21298211/200221604-cb0107bd-4d4d-4cf4-95ce-694a12dd8c96.png)

Now I don't know about you guys, but I had a hard time figuring out the error. Especially when the source code looks like this (without any VS extension installed).
![image](https://user-images.githubusercontent.com/21298211/200221703-8b528df0-cf39-435a-8a98-2096a3e0cc28.png)

Only after installing the yaml extension by RedHat could I see that the error was that \t (tab) was used instead of spaces. But in VSCode, my tab key auto-converts to spaces.

Please make this error message more explicit so I don't end up chasing a goose.

**Comment by ossanna16** [comment]
Hi there @elibroftw and welcome to our community! Thank you for reporting this!
